autoload/neomake/makers/ft/rst.vim

```diff
@@ -23,3 +23,19 @@ function! neomake#makers#ft#rst#rstcheck()
             \ '%E%f:%l: (SEVERE/4) %m',
         \ }
 endfunction
+
+" TODO: determine proper path of current .rst file upwards in sear for conf.py file, use that instead of "docs"
```

Comment:
Wouldn't be worth having it for just one add-on but if you do end up adding something usable, i'd gladly use it. (See also https://github.com/vim-syntastic/syntastic/blob/0f9fa8a0c1ccb61544d370a53a5efe8184b3c1e1/autoload/syntastic/util.vim#L253)